library/src/actions/jwt/jwt.ts

```diff
@@ -0,0 +1,193 @@
+import type {
+  BaseIssue,
+  BaseValidation,
+  ErrorMessage,
+} from '../../types/index.ts';
+import { _addIssue } from '../../utils/index.ts';
+
+const POSSIBLE_JWT_REGEX = /^(?:[\w-]+\.){2}[\w-]+$/u;
+
+// https://datatracker.ietf.org/doc/html/rfc7518#section-3.1
+type JwtAlgorithm =
+  | 'HS256'
+  | 'HS384'
+  | 'HS512'
+  | 'RS256'
+  | 'RS384'
+  | 'RS512'
+  | 'ES256'
+  | 'ES384'
+  | 'ES512'
+  | 'PS256'
+  | 'PS384'
+  | 'PS512'
+  | 'none';
+
+function isValidJwt(
```

Comment:
Would you recommend inlining this function in the `requirement` function of the implementation? Or is this okay? I think this is better for readability.

Reply:
I think I would implement it similar to how `creditCard` is implemented.